jwst-masterclass/base-environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - manics
+dependencies:
+  - websockify
+  - pip:
+    - https://github.com/jupyterhub/jupyter-server-proxy/archive/0e67e1afd0bab1342443f13bd147a2f8c682e9e0.zip
```

Comment:
based on https://github.com/berkeley-dsep-infra/datahub/pull/1140/files#diff-bcf82d4102816bc5450e2639a53c2240R7, I think you can just add 'jupyter-desktop-server' to the list here under pip, and remove the jupyter_desktop folder (and files related to that) entirely. This will make sure you get all the latest updates, and not duplicate code

Reply:
It would also mean you don't need postBuild